pkg/apis/apis.go

```diff
@@ -0,0 +1,36 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
I prefer the following format:
```
/*
Copyright 2024 The CloudPilot AI Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

Reply:
A suggestion： `Copyright 2024 The Kubernetes Authors.` to kubernetes-sig for future.

Reply:
This is not a sig project, in the future, if we move this project to sig, we can modify this

Reply:
Maybe leaving blank space is also a solution, like AWS.

Reply:
+1

Reply:
for azure provider: https://github.com/Azure/karpenter-provider-azure/blob/846ef96155e5e402e27a0ac099db11b0243d5d01/pkg/cache/cache.go#L1C1-L15C3


It's a better option to keep CloudPilot AI